docs/guide/snaps.md

```diff
@@ -30,55 +31,38 @@ Among other things, SES allows us to restrict access to global JavaScript APIs a
-- git
-- [yarn v1](https://classic.yarnpkg.com)
-- Up-to-date Chromium or Firefox browser
-
-### Installing MetaMask Flask
-
-[MetaMask Flask](https://metamask.io/flask) is a separate extension for developers that provides access to additional unstable APIs. The goal of Flask is to maximize developer control, so that we can learn the full extent of what developers want to do with MetaMask and incorporate those lessons into the stable MetaMask distribution. Snaps is the first feature to be released on Flask.
-
-- First, in a Firefox or Chromium browser, create a new browser profile or disable any existing installed versions of MetaMask.
-  - Running multiple instances of MetaMask in the same browser profile will break dapp interactions.
-- Next, download [Flask](https://metamask.io/flask) and add it your chosen browser.
+### Install MetaMask Flask
 
-### Serving a Snap to Your Local Environment
+To interact with (your) Snaps, you will need to install [MetaMask Flask](https://metamask.io/flask/), a canary distribution for developers that provides access to upcoming features. 
 
-With Flask in your browser, it's time to plug something into it. If you don't want to download any code for the moment, check out the example snap [deployed here](https://metamask.github.io/template-snap/) and follow the instructions below under 'Interacting with the Snap'.
+Make sure to install Flask in a new browser profile or disable any existing installed versions of MetaMask. Running multiple instances of MetaMask in the same browser profile will break dapp interactions.
 
-If you're ready to jump in and see how a Snap looks under the hood, follow the steps below!
+### Quick start using our template
+Get started using our template built with TypeScript and React. Clone the repository via the command line or [via GitHub](https://github.com/MetaMask/template-snap-monorepo/generate).
+```shell
+git clone git@github.com:MetaMask/template-snap-monorepo.git    
+```
 
-Open up a new terminal window, and do the following:
 
-- Clone the [template snap](https://github.com/MetaMask/template-snap) repo.
-- `cd` into the repo, and run `nvm use`.
-  - This will install and/or use the recommended NVM version from `.nvmrc`.
-- Run `yarn install && yarn build:clean && yarn serve`.
-  1. This first runs the commands needed to install dependencies.
-  1. If there is a `dist/` folder, it is then cleaned before the build command is run.
-  1. Finally, the dapp is served on `http://localhost:8080`.
+Setup the development environment serving both the frontend and the snap: 
+```shell
+yarn install && yarn start
+```
 
-Your terminal window should now be serving the example snap locally. Time to see the functionality in action!
+### Get started by editing `index.js` in the `snap` folder
+Add your APIs and features to MetaMask.
```

Comment:
It would be good to expand upon this slightly. Or, perhaps merge this with "Where to go from here"? It's sort of like we're saying "Now it's your turn!", but the reader might ask, "Great, but what do I do?"

Reply:
I agree completely; do we have an example, internal or external, of how you would add your API or feature? Obviously if someone is attempting this, they are a fairly experienced engineer, but any kind of schema or best practices are probably a good idea.

Reply:
Maybe the next section is our answer--say, "If you're looking for examples or further guidance as to how this is done, follow the steps below."